APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,212 @@
+
+This de facto checklist implies that a more transparent and understandable
+policy would be to make this checklist formal and completely public, and use it
+directly as the formal review criteria - i.e., a rubric.  These criteria will be
+laid out explicitly in a document, with various levels defined for each criteria
+(generally corresponding to "good", "acceptable", and "unacceptable"). This
+will make the review process much faster and (eventually) open the review
+process beyond the coordination committee, thereby address the concerns from the
+"Motivation" section.
+
+
+A community review process for affiliated packages
+==================================================
+
+On a longer term horizon, a criteria-based review checklist will allow the
+review process to change to a more "peer-review" model.  That is, instead of
+every package being vetted by the coordination committee, the committee can act
+more as "editors", sometimes offering the review opportunity to other members of
+the Astropy community.  While the details of this process will not be outlined
+here, as they by necessity should be flexible enough to accomodate the
+experiences of the first few reviewers, this APE sets this as the eventual
+*goal* for the affiliated package review process.
+
+
+Branches and pull requests
+--------------------------
+
+N/A
+
+
+Implementation
+--------------
+
+The changes outlined in this APE will be implemented as three distinct efforts:
+
+* The new package review criteria have already been implemented by the
+  coordination committee and reviewed by the community (Available in Google doc
+  form `here <https://docs.google.com/document/d/15PJf2PROXMa7xwTDvWnjXB_9KNuO2Ia4_kkxJ7MPazE/edit?usp=sharing>`_.
+  These criteria will need to be translated to HTML and put on the affiliated
+  package instructions section of the Astropy website.  The affiliated package
+  listing page will also need to be updated for these new criteria - a concept
+  for this new site is `viewable here <example_affilpkg_page.png>`_.
+* The Astropy web site (and to a lesser extent, affiliated package-related docs)
+  will need to be updated to reflect the existence of the coordinated
+  packages.  Additionally, the roles web site will need to be populated with the
+  new roles for the coordinated packages (and the role descriptions).  This
+  requires relatively little effort and can be completed by the Astropy web site
+  maintainers upon acceptance of this APE.
+* A "peer-review" process will need to be set up to enable reviews by community
+  members beyond the coordination committee.  While such processes exist in
+  other contexts (e.g. the
+  `Journal of Open Source Software <http://joss.theoj.org/>`_ and
+  `Ropensci <https://ropensci.org/>`_) that may have tools that can be re-used,
+  it is likely that setting up this process will take some time - this APE does
+  not outline the details of this implementation, but rather gives the broad
+  goals.
```

Comment:
Do we need to be that complicated? I mean, mow many reviews are there? Taking the current list of affiliated packages it looks like about 1-2 per months. Can't we adopt the same process and leave the implementation to email and a repro that keeps all results that the editors want to open source (just paste reviews from emails into a text file)?
While a more general approach might be better in the long term, are we sure that it's worth the initial investment at this point? Astropy has done good work in building tools that are used routinely and sometimes outside of astropy, too (e.g. ci-helpers), but we've also occasional wasted quite some time overthinking a problem so that we now have a technically brilliant solution that took a long time to build and is now hard to maintain (e.g. the io.ascii readers) or had to be simplified later (the intii8al concepts for nddata). It sounds to me it might be easier to peer=review with a low entry cost solution (email) and see how it goes.

Reply:
I agree with @hamogu here. Would suggest to explicitly write that an initial trial will be done simply using e-mail, with the editors responsible for putting the results up in a usable form. Then, after some experience, it can be decided whether something more complicated is useful, or whether the instructions to the referees need to be sharpened, etc.